README.md

```diff
@@ -94,6 +94,7 @@ Launcher props:
 | *agentProfile | object | Represents your product or service's customer service agent. Fields: teamName, imageUrl|
 | onMessageWasSent | function(message) | Called when a message a message is sent with a message object as an argument. |
 | messageList | [message] | An array of message objects to be rendered as a conversation. |
+| toggle | boolean | Toggle this boolean to open chat window. Allows customization for event driven opening of the chat. |
```

Comment:
hmm, the 'toggle' boolean will not indicate the current state, componentDidUpdate just checks for a change and if there is a change, it will open the chat window. it does not close the chat window.

I made it this way for the use cases of, for example, you click on someone's profile and want their chat window to open. I will change the variable name but let me know if you do not like the set up

Reply:
This particular thing got resolved by #34 -- so we shouldn't need to merge this in. I'll close for now, let me know if I've missed something.